planterbox_webdriver/util.py

```diff
@@ -285,4 +285,4 @@ def submit_form(element):
             "e.initEvent('submit', true, true);"
             "arguments[0].dispatchEvent(e);", form)
     else:
-        element.submit()
+        element.click()
```

Comment:
The problem here is that `._w3c` doesn't exist anymore - `AttributeError: 'WebElement' object has no attribute '_w3c'` - so we can't switch between strategies based on that. `.click()` seems like it won't work here, because this takes the form, not the submit button. I think we just want to eliminate the check and just have lines 282-286 as the entire body of this function.

Reply:
submit has been deprecated so `.click()` is what's left. I will research about this `._w3c` I am not familiar with it

Reply:
Please do not research.

Reply:
The `._w3c` check here was a hack for determining whether `.submit()` would work or not; with our new minimum `selenium` version dependency, we _know_ it won't, so we should _only_ use the solution from lines 282-286.